src/runtime/virtcontainers/sandbox.go

```diff
@@ -1161,7 +1161,6 @@ func (s *Sandbox) startVM(ctx context.Context) (err error) {
 	}); err != nil {
 		return err
 	}
-
```

Comment:
This file should be removed from this commit.

Reply:
?